README.md

```diff
@@ -90,14 +117,38 @@ This folder contains examples that illustrate the usage of individual features o
 
     This tutorial illustrates how Amazon Braket integrates with Amazon EventBridge for event-based processing. In the tutorial, you will learn how to configure Amazon Braket and Amazon Eventbridge to receive text notification about quantum task completions on your phone. Of course, EventBridge also allows you to build full, event-driven applications based on events emitted by Amazon Braket.
 
+* [**Noise Models on Amazon Braket**](examples/braket_features/Noise_models/Noise_models_on_Amazon_Braket.ipynb)
+
+    This tutorial shows how to create noise models containing different types of noise and instructions for how to apply the noise to a circuit. A noise model encapsulates the assumptions on quantum noise channels and how they act on a given circuit. Simulating this noisy circuit gives information about much the noise impacts the results of the quantum computation. By incrementally adjusting the noise model, the impact of noise can be understood on a variety of quantum algorithms.
+
+* [**Noise Models on Rigetti**](examples/braket_features/Noise_models/Noise_models_on_Rigetti.ipynb)
```

Comment:
Honestly, I was trying to avoid using the specific device name so that we don't forget to update it, but I think we have lots of other places where we would need to update it anyways...

Reply:
Yeah. I made it Rigetti's Aspen device. Do you think that's OK now?